src/SlashCommands.tsx

```diff
@@ -1129,6 +1130,23 @@ export const Commands = [
         },
         category: CommandCategories.advanced,
     }),
+    new Command({
+        command: "tovirtual",
+        description: _td("Switches to this room's virtual room, if it has one"),
+        category: CommandCategories.advanced,
+        runFn: (roomId) => {
+            return success((async () => {
+                const room = await VoipUserMapper.sharedInstance().getVirtualRoomForRoom(roomId);
+                if (!room) throw _t("No virtual room for this room");
```

Comment:
throwing a string is a bit different from what I'd expect. It looks like other commands can't really decide on a pattern, but possibly leaning more towards `throw newTranslatableError()` would be good?